qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -573,8 +573,12 @@ def _qubit_operator_to_terms(qubit_operator, wires=None):
             (
                 coef,
                 qml.operation.Tensor(*[xyz2pauli[q[1]](wires=wires[q[0]]) for q in term])
-                if term
-                else qml.operation.Tensor(qml.Identity(wires[0]))
+                if len(term) > 1
+                else (
+                    xyz2pauli[term[0][1]](wires=wires[term[0][0]])
+                    if len(term) == 1
```

Comment:
Is the check necessary can `len(term)`be equal to 0?

Reply:
Yes
![image](https://user-images.githubusercontent.com/48225584/129405201-1fa36e19-7753-46e4-85bd-d5766432f459.png)